data/mods/ssb/statuses.ts

```diff
@@ -258,12 +258,13 @@ export const BattleStatuses: {[k: string]: ModdedPureEffectData} = {
 	},
 	epicnikolai: {
 		noCopy: true,
-		onStart() {
+		onStart(source) {
 			this.add(`c|${getName('EpicNikolai')}|I never give up until I get something right, which means destroying you ☜(ﾟヮﾟ☜)`);
 			if (source.species.id !== 'garchompmega' || source.illusion) return;
 			this.add('-start', source, 'typechange', source.types.join('/'), '[silent]');
 		},
 		onSwitchOut() {
+			// @ts-ignore
 			this.add(`c|${getName('EpicNikolai')}|This wasn't as fun as I thought it would be, I'm out ¯_( ͡~ ͜ʖ ͡°)_/¯`);
 		},
```

Comment:
```suggestion
			this.add(`c|${getName('EpicNikolai')}|This wasn't as fun as I thought it would be, I'm out ¯_( ͡~ ͜ʖ ͡°)_/¯`); // eslint-disable-line no-irregular-whitespace
		},
```